core/src/main/java/ar/com/wolox/wolmo/core/fragment/WolmoFragmentHandler.java

```diff
@@ -120,24 +124,9 @@ View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
     @SuppressWarnings("unchecked")
     void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
```

Comment:
Add the `@NonNull` to the `View view` parameter. It was added in one of the latest Android API updates. It's nice because, if this method is called, the `Fragment` *has to have* a `View`.